pub/css/main.css

```diff
@@ -0,0 +1,371 @@
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding: 0;
+}
+
+audio,
+canvas,
+img,
+video {
+    vertical-align: middle;
+}
+
+fieldset {
+    border: 0;
+    margin: 0;
+    padding: 0;
+}
+
+textarea {
+    resize: vertical;
+}
+
+.browsehappy {
+    margin: 0.2em 0;
+    background: #ccc;
+    color: #000;
+    padding: 0.2em 0;
+}
+
+
+/* ==========================================================================
+   Author's custom styles
+   ========================================================================== */
+
+/*Container Style*/
+.container {
+width: 80%; margin: auto;
+}
+
+/*Header Style*/
+header h1 {
+    text-align: center;
+    font-family: 'Droid Sans', sans-serif;
+    color: #46A4DA;
+    font-size: 2em;
+}
+
+header i {
+    color: #46A4DA;
+    border: 1ex solid #46A4DA;
+    border-radius: 50%;
+    padding: 1ex;
+    margin-bottom: 5ex;
+}
+
+header div {
+    width: 30%;
+    text-align: center;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.timeline {
+    margin: 0 auto; padding: 0;
+    list-style: none;
+    padding-top: 3em;
+}
+
+.timeline-post article {
+    display: block;
+    position: relative;
+    padding: 1ex 2em;
+}
+
+.timeline-post {
+    margin-bottom: 6em;
+}
+
+time small {
+    color: darkgray;
+}
+
+time big {
+    font-size: 1.5em
+}
+
+.timeline-post:nth-child(odd) {
+    background-color: #6CBFEE;
+    border-radius: 1em;
+}
+
+.timeline-post:nth-child(even) {
+    background-color: #46A4DA;
+    border-radius: 1em;
+}
+
+.timeline-post article {
+    font-family: 'Droid Sans', sans-serif;
+    margin-bottom: 2em;
+    color: #FFFFFF;
+}
+
+.timeline time {
+    float: left;
+    display: block;
+    margin-top: -4em;
+    position: absolute;
+}
+
+.light  {
+    color: #46A4DA;
+}
+
+.dark  {
+    color: #6CBFEE;
+}
+
+
+.icon {
+    float: right;
+    padding: 1ex;
+    position: relative;
+    top: -3.5em;
+    border: 1ex solid #6CBFEE;
+    background-color: #46A4DA;
+    border-radius: 50%;
+    color: #FFFFFF;
+    line-height: 1em;
+    font-size: 1.2em;
+
+
+}
+
+
+article h3 {
+    font-weight: 700;
+    border-bottom: 1px solid #FFFFFF;
+    font-size: 1.5em;
+}
+
+
+
+/* ==========================================================================
+   Media Queries
+   ========================================================================== */
+
+@media only screen and (min-width: 35em) {
+
+}
+
+@media print,
+       (-o-min-device-pixel-ratio: 5/4),
+       (-webkit-min-device-pixel-ratio: 1.25),
+       (min-resolution: 120dpi) {
+
+}
+
+
+/* Medium Breakpoint----------- */
+@media (min-width : 768px) {
+
+header h1 {
+    font-size: 3em;
+}
+
+.timeline {
+    padding-left: 2em;
+    border-left: 1em solid #46A4DA;
+    width: 70%;
+    font-size: 1.2em;
+    padding-top: 0
+}
+
+
+
+.icon {
+    float: left;
+    right: 3.6em;
+    top: -1em;
+    margin-right: -4em
+}
+
+time {
+    position: absolute;
+    margin-left: -8.5em;
+}
+
+
+
+
+
+
```

Comment:
So you don't end up forgetting to close the block, like you did here. Like with HTML, I generally put a comment after the closing bracket to remind me what it's supposed to close.